ubuntu1804_rocm/Dockerfile

```diff
@@ -0,0 +1,167 @@
+FROM rocm/dev-ubuntu-18.04:4.0.1
+
+LABEL description="Ubuntu 18.04 with Acts and HIP dependencies"
+LABEL maintainer="Paul Gessinger <paul.gessinger@cern.ch"
```

Comment:
Do you want to put your name here?

Reply:
I could. I just thought that since this is 99.9% same as the CUDA image's configuration, I definitely don't deserve to take credit for writing it. And I'm also happy to let you update it with new versions of ROOT, Boost, etc. later on.

So all in all, I'm happy to leave you to maintain it after it was added. :stuck_out_tongue: But if you'd like to share the maintenance effort of these images, I could take part in it.